changelog/fix-learning-mode-template-on-wpcom

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Fix some lesson styling issues when Gutenberg is enabled
```

Comment:
Maybe we can update this, as with WP 6.3, these issues occur even without Gutenberg

Reply:
[285a308](https://github.com/Automattic/sensei/pull/7054/commits/285a308d0683fc74a3221e525d08911efc5fa49f)